spec/features/delete_draft_asset_manager_down_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+RSpec.feature "Delete draft with Asset Manager down" do
+  scenario "Delete draft with Asset Manager down" do
```

Comment:
I feel this should be separate as it's a different scenario from the Publishing API one - otherwise we'd have to reset the document state half way through the scenario.